examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>From the introduction before the bullet points, I understand that all these variables are part of the ArviZ schema and not all are. I would use something like "Some of the sample statistics used by NUTS are renamed when converting to <code>InferenceData</code>  to follow [ArviZ's naming convention], whereas some others are specific to PyMC3 and keep their internal PyMC3 name in the resulting <code>InferenceData</code>  object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>